[DOCS] DOC-371 update the getting started tutorial for data assistants

Rachel-Reverie opened the pull request:
Changes proposed in this pull request:
- update the getting started tutorial for data assistants


### Definition of Done

- [X] I have made corresponding changes to the documentation

Rachel-Reverie committed:
DOC-371
- update the getting started tutorial for data assistants

ghost commented:
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=6024&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=6024&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)
</details>

[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/great-expectations/great_expectations/6024/0270e054/be6f5f21d5e814ab8081de4c785e702dac041d6b.svg)](https://app.codesee.io/r/reviews?pr=6024&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

Rachel-Reverie committed:
DOC-371
- typo fix

Rachel-Reverie committed:
DOC-371
- update command for the CLI in the guide on using the onboarding data assistant

donaldheppner committed:
updated screenshot to match the latest notebook

donaldheppner committed:
replaced profiler with Data Assistant

Shinnnyshinshin commented on docs/guides/expectations/components_how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data/_use_the_builtin_help_to_review_the_clis_suite_new_optional_flags.mdx:
````diff
@@ -1,5 +1,5 @@
 
-To check the syntax for optional flags that the `great_expeftation suite new` CLI command accepts, you can always run the following command in the root directory of your project (where the `great_expectations init` command created the `great_expectations` subdirectory:
+To check the syntax for optional flags that the `great_expectations suite new` CLI command accepts, you can always run the following command in the root directory of your project (where the `great_expectations init` command created the `great_expectations` subdirectory:
 
 ```bash title="Terminal input"
````
😓

Shinnnyshinshin reviewed (approved):
Looks wonderful @Rachel-Reverie thank you 🙇🏼

donaldheppner committed:
Merge branch 'develop' into d/doc-371/getting_started_tutorial_update_for_data_assistants
